modules/linear_boltzmann_solvers/lbs_problem/lbs_problem.cc

```diff
@@ -1023,26 +1019,7 @@ LBSProblem::InitializeParrays()
     {
       if (not face.has_neighbor)
       {
-        Vector3& n = face.normal;
-
-        int boundary_id = -1;
-        if (n.Dot(ihat) < -0.999)
-          boundary_id = XMIN;
-        else if (n.Dot(ihat) > 0.999)
-          boundary_id = XMAX;
-        else if (n.Dot(jhat) < -0.999)
-          boundary_id = YMIN;
-        else if (n.Dot(jhat) > 0.999)
-          boundary_id = YMAX;
-        else if (n.Dot(khat) < -0.999)
-          boundary_id = ZMIN;
-        else if (n.Dot(khat) > 0.999)
-          boundary_id = ZMAX;
-
-        if (boundary_id >= 0)
-          face.neighbor_id = boundary_id;
```

Comment:
Not needed anymore?

Reply:
While this PR removes this code, it does preserves this functionality. It is just now located in `MeshContinuum::SetOrthogonalBoundaries()` and must be explicitly asked for by a user.

There is a slight change thou. We are not setting all of `[x|y|z][min|max]`, we only set the ones that make sense for given spatial dimension.